hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/AbstractContractAppendTest.java

```diff
@@ -75,14 +83,11 @@ public void testBuilderAppendToEmptyFile() throws Throwable {
 
   @Test
   public void testAppendNonexistentFile() throws Throwable {
+    //expected
     try {
-      FSDataOutputStream out = getFileSystem().append(target);
-      //got here: trouble
-      out.close();
-      fail("expected a failure");
-    } catch (Exception e) {
-      //expected
-      handleExpectedException(e);
+      getFileSystem().append(target).close();
+    } catch (Exception ex) {
```

Comment:
this removes the entire failure path of the test: The assertion that appending to a nonexistent file raises an error. Why did you make this change?

Reply:
This should be fixed now after cleaning the commit and revert part in hadoop-common.